.github/workflows/ros2_build_test/entrypoint.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh -l
+. /opt/ros/humble/setup.sh
+
+colcon build --continue-on-error --packages-skip alica_ros_proxy alica_tracing alica_ros_turtlesim alica_tests supplementary_tests alica_msgs 2>&1 | tee result.txt
```

Comment:
I have confirmed that the return value of `colcon build` is indeed 0 if it succeeds and nonzero otherwise, so there is no need to grep at the results

Reply:
Cool! If there was a documentation page that listed that somewhere please do send it over so I can see if I missed anything else!

Reply:
I just ran it and then did `echo $?` to see the return value